app/assets/javascripts/components/autocomplete.js

```diff
@@ -40,4 +40,37 @@ if ($multiselects) {
     })
   })
 }
+
+var $customTemplateAutocomplete = document.querySelector('[data-module="autocomplete-custom-template"]')
+if ($customTemplateAutocomplete) {
+
+  new accessibleAutocomplete.enhanceSelectElement({
+    selectElement: $customTemplateAutocomplete,
+    minLength: 3,
+    showNoOptionsFound: false,
+    autoselect: false,
+    defaultValue: '',
+    templates: {
+      inputValue: function (result) {
+        if (result) {
+          return result.split(' - ')[0]
+        }
+      },
+      suggestion: function (result) {
+        if (result) {
+          var resultItems = result.split(' - ')
```

Comment:
Are we sure this character sequence won't occur in what should be the first entry?

Reply:
Yes, we tested and for now it's safe to use it. It's only present in one organisation which doesn't have any contacts.